ui/src/alerting/components/builder/CheckMessageCard.tsx

```diff
@@ -74,6 +74,15 @@ const CheckMessageCard: FC<Props> = ({check, onUpdateTimeMachineCheck}) => {
             <code>{'${string(v: r.numericColumn)}'}</code> Functions can be used{' '}
             as well
           </p>
+          <p>
+            Need help? Check out the Status Message Template{' '}
+            <a
+              href="https://github.com/influxdata/docs-v2/blob/monitor-alert/content/v2.0/monitor-alert/checks/create.md#flux-only-interpolates-string-values"
```

Comment:
Link should be https://v2.docs.influxdata.com/v2.0/monitor-alert/checks/create/#flux-only-interpolates-string-values